cfme/intelligence/reports/menus.py

```diff
@@ -1,160 +1,183 @@
 # -*- coding: utf-8 -*-
 """Module handling report menus contents"""
 from contextlib import contextmanager
-
-from . import Report
-from cfme.fixtures import pytest_selenium as sel
-from cfme.intelligence.reports.ui_elements import FolderManager
-from cfme.web_ui import Region, BootstrapTreeview, Tree, accordion, form_buttons
-from cfme.web_ui.multibox import MultiBoxSelect
-from utils import version
-from utils.appliance.implementations.ui import navigate_to
-from utils.log import logger
-
-
-def reports_tree():
-    if version.current_version() >= '5.7':
-        return BootstrapTreeview("menu_roles_treebox")
-    else:
-        return Tree("//div[@id='menu_roles_treebox']/ul")
-
-
-manager = FolderManager("//div[@id='folder_lists']/table")
-report_select = MultiBoxSelect(
-    "//select[@id='available_reports']",
-    "//select[@id='selected_reports']",
-    "//a[@title='Move selected reports right']/img",
-    "//a[@title='Move selected reports left']/img",
-)
-
-buttons = Region(locators=dict(
-    commit="//a[@title='Commit report management changes']/img",
-    discard="//a[@title='Discard report management changes']/img",
-))
-
-default_button = form_buttons.FormButton("Reset All menus to CFME defaults")
-
-
-def go_to_group(group_name):
-    navigate_to(Report, 'EditReportMenus')
-    accordion.tree("Edit Report Menus", "All EVM Groups", group_name)
-
-
-def get_folders(group):
-    """Returns list of folders for given user group.
-
-    Args:
-        group: User group to check.
-    """
-    go_to_group(group)
-    reports_tree().click_path("Top Level")
-    return manager.fields
-
-
-def get_subfolders(group, folder):
-    """Returns list of sub-folders for given user group and folder.
-
-    Args:
-        group: User group to check.
-        folder: Folder to read.
+from utils.appliance import Navigatable
+from utils.appliance.implementations.ui import navigate_to, navigator, CFMENavigateStep
+from widgetastic.widget import Text
+from widgetastic_manageiq import ManageIQTree, FolderManager
+from widgetastic_patternfly import Button
+from navmazing import NavigateToAttribute
+
+from . import CloudIntelReportsView, ReportsMultiBoxSelect
+
+
+class EditReportMenusView(CloudIntelReportsView):
+    title = Text("#explorer_title_text")
+    reports_tree = ManageIQTree("menu_roles_treebox")
+    # Buttons
+    save_button = Button("Save")
```

Comment:
@mshriver we had a discussion about that. I've been using such names for the buttons since widgetastic conversion started. Besides I follow @mfalesni [Automate](https://github.com/ManageIQ/integration_tests/tree/master/cfme/automate) code.